CMakeLists.txt

```diff
@@ -13,21 +18,22 @@ set(main_exe_name RigelEngine)
 # Project settings
 ###############################################################################
 
-option(USE_GL_ES "Use OpenGL ES instead of regular OpenGL" OFF)
+option(USE_GL_ES "Use OpenGL ES instead of regular OpenGL" ON)
```

Comment:
We should have some code to set the GL ES option to ON if building with Emscripten. That way, we can keep the default value at `OFF`.

Reply:
there exists a __EMSCRIPTEN__ guard which I am planning to use

Reply:
For this, we can do it in CMake:

```
if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Emscripten")
  set(USE_GL_ES ON)
endif()
```